### DISCUSSION TITLE
Database connection

### DISCUSSION THREAD

**Discussion by igmltd** [rapidez/rapidez#88] [Q&A]
Hi, I tried installing it into an existing project, does the magento database need to be the same as the laravel database?

i get this error when running validate:

```
SQLSTATE[42S02]: Base table or view not found: 1146 Table 'muweb.core_config_data' doesn't exist (Connection: mariadb, SQL: select `value` from `core_config_data` where `path` = catalog/frontend/flat_catalog_product and ((`scope` = stores and `scope_id` = 1) or (`scope` = websites and `scope_id` is null) or (`scope` = default and `scope_id` = 0)) order by FIELD(scope, "stores", "websites", "default") ASC limit 1)

  at vendor/laravel/framework/src/Illuminate/Database/Connection.php:825
    821▕                     $this->getName(), $query, $this->prepareBindings($bindings), $e
    822▕                 );
    823▕             }
    824▕ 
  ➜ 825▕             throw new QueryException(
    826▕                 $this->getName(), $query, $this->prepareBindings($bindings), $e
    827▕             );
    828▕         }
    829▕     }

      +36 vendor frames 

  37  artisan:13
      Illuminate\Foundation\Application::handleCommand(Object(Symfony\Component\Console\Input\ArgvInput))
   ```   
      
      it seems to be looking for my laravel database. my magento database is separate,

      
      
      please kindly advise, thanks.

**Comment by royduin** [comment]
The easiest way is to change your current application models and include a different database connection so all Rapidez models can use the default one, see: https://laravel.com/docs/master/eloquent#database-connections

I'll make an internal note for this, alternatively you can publish the Rapidez config and overwrite the models to include a different database connection, see: https://docs.rapidez.io/2.x/extending.html#models

**Reply by igmltd** [reply]
That’s perfect.

Thank you for taking time out to reply - very much appreciated.
